README.md

```diff
@@ -66,6 +66,9 @@ The following platforms will eventually be included in the "Core" set when prope
   * ARMv8 64-bit (Linux, one day, when suitable hardware & OS is available)
   * Ubuntu 14.10 (Utopic Unicorn) 64-bit (preferably prior to official release)
   * Mac OS X 10.10 (Yosemite) + XCode 6
```

Comment:
Agreed it seemed a bit stale in some places.  @jbergstroem if there are other updates I should pull into this PR just let me know